features/step_definitions/web_steps.rb

```diff
@@ -21,6 +21,7 @@ def with_scope(locator)
 
 When /^(?:|I )go to (.+)$/ do |page_name|
   visit path_to(page_name)
+  wait_for_requests
```

Comment:
Are there AJAX call in this feature? I'd prefer not waiting for the requests for every steps where "I go to"